dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -49,7 +49,7 @@ public CultureInfo Culture
     /// <summary>
     /// Executes functions using the current resources loaded in the context
     /// </summary>
-    public IFunctionRunner Runner { get; }
+    public Kernel Runner { get; }
```

Comment:
I assume we're not renaming this because we expect the whole class to disappear soon.

Reply:
Yep, right - no need to rename since it'll be removed soon.